docs/saas.rst

```diff
@@ -4,20 +4,115 @@
+as a service of Web 3.0 where you can interact with it through the browser, or an API, if you so desire.
 
-Please read the warnings below, as you accept some inherent risks when using xOpera-SaaS
+Please read the `warnings <xopera_saas_warnings_>`_ at the bottom, as you accept some inherent risks when using xOpera SaaS.
+
+=================
+Browser interface
+=================
 
 Using the browser version is straightforward.
+Let's go through the basic workflow, where you:
+
+1. Have credentials you need to define prior to deployment.
+
+2. Author a new workspace to contain your project.
+
+3. Register your credentials to be available in the workspace.
+
+4. Use the browser to create a new xOpera project from a CSAR.
+
+5. Have to specify which service template and inputs you are using, then validate them.
+
+6. In the end, deploy the project.
+
+.. _xopera_saas_credentials:
+
+.. figure:: /images/xopera-saas-credentials.png
+    :target: _images/xopera-saas-credentials.png
+    :width: 95%
+    :align: center
+
+    The credentials screen.
+
+The first thing we need to do is create whatever credentials are necessary for your deployment to run.
+For example, these are your cloud provider credentials, SSH public keys, among others.
+The way they are provided is through files - with each credential, you declare a file (and contents) that will be
+present in your project when you create it.
+
+.. warning::
+
+    Credentials need to be created and assigned to workspaces before a project is created.
+    Newly defined credentials do not have an effect on existing projects.
+
+Next, let's create a workspace to contain our projects.
+
+.. _xopera_saas_workspaces:
+
+.. figure:: /images/xopera-saas-workspaces.png
+    :target: _images/xopera-saas-workspaces.png
+    :width: 95%
+    :align: center
 
-Using the xOpera SaaS API through ``curl``::
+    Manage numerous workspaces directly from your browser.
+
+Creating one is simple, you just need to choose a name.
+You are assigned owner privileges automatically, and you can share this workspace with other users, who can then
+also create projects in it.
+In :numref:`xopera_saas_workspaces`, the `DemoWorkspace` projects is shared with us, which we can determine by looking
+at the *Ownership* column.
+Sharing projects is done through the dropdown menu on the right, by clicking on the kebab icon.
+
+The next thing we need to do is to assign the credentials we created in the previous step to this workspace.
+This is the only way they are applied to projects subsequently created within this workspace.
+As with sharing projects, this is done through the dropdown on the right of each workspace's row.
+When you apply a credential, this is reflected in the list of workspaces.
+
+All that is left is to create and deploy a project.
+To do this, click the *Add Project* button, choose a name and select your CSAR file.
```

Comment:
Hey, I just found something interesting- Sphinx has [`:guilabel:`](https://devopstutodoc.readthedocs.io/en/0.3.0/documentation/doc_generators/sphinx/rest_sphinx/markup/inline/guilabel.html) role designed for buttons and it looks nice in the RTD theme, so if it's not too much consider adding this role to all clickable buttons that you mention in the docs. 

![image](https://user-images.githubusercontent.com/41727816/101913758-b79b4580-3bc3-11eb-9eeb-eed14d4c945a.png)

Reply:
Wow!